modules/nextflow/src/main/groovy/nextflow/processor/TaskConfig.groovy

```diff
@@ -377,6 +377,17 @@ class TaskConfig extends LazyMap implements Cloneable {
         return get('container')
     }
 
+    Arch getArch() {
```

Comment:
Similarly, I think this should be `getArchitecture()`

Reply:
done!